utils/rtty/files/rtty.init

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=99
+
+BIN=/usr/sbin/rtty
+
+invalid=
+ifname=
+id=
+description=
+host=
+port=
+
+device_config() {
+    local cfg="$1"
+
+    uci_validate_section rtty device "${1}" \
+        'interface:string:br-lan' \
+        'id:and(string,maxlength(63))' \
+        'description:and(string,maxlength(126))' || invalid=1
```

Comment:
`interface` could be validated as `uci("network", "@interface")`

`maxlength(NN)` is sufficient (`string` without min max limit is an always true predicate)

Reply:
Fixed.